src/views/payment-sheet-views/paypal-view.js

```diff
@@ -61,7 +61,11 @@ PayPalView.prototype._initialize = function () {
       onError: reportError
     };
 
-    paypal.Button.render(paypalCheckoutConfig, '[data-braintree-id="paypal-button"]').then(function () {
+    if (merchantConfiguration.locale) {
```

Comment:
I'm personally okay with leaving it how it is until someone comes up with a valid use case for differing locales.